openapis/swagger.yaml

```diff
@@ -149,6 +149,90 @@ paths:
         - cdis_auth:
```

Comment:
let's remove this yaml file, should just autogenerate it from the json on client side and no need to keep in github